docs/installation.mdx

```diff
@@ -123,4 +123,16 @@ Version:        1.0.0
 Go version:     go1.20.2
 Git commit:     b58e7b910ca556973d111e9bd734a71baef03db2
 Git tree state: clean
-```
\ No newline at end of file
+```
+
+## To validate the checksum of ORAS CLI binaries:
+
+You can download all of the artifacts uploaded to the new [GitHub release](https://github.com/oras-project/oras/releases) locally.
+
+The checksum can be performed using the following command:
+
+```
+shasum -c oras_${version}_checksums.txt
+```
+
```

Comment:
Apart from checksum file, there should be a GPG key verification process as I explained in https://github.com/oras-project/oras-www/issues/69.